database/jkpgcity-tables.sql

```diff
@@ -0,0 +1,156 @@
+USE jkpg;
+
+-- Create store table
+
+DROP TABLE IF EXISTS STORES; -- We drop the table if it exists to avoid errors this might change in the future
+
+CREATE TABLE STORES (
```

Comment:
If the variables inside are lowercase, then the table name should also be lowercase. Choose one and do not mix them up, I would prefer lowercase on everything.